dashboard/utils.py

```diff
@@ -239,6 +239,26 @@ def get_payments_count_for_course(self, course):
             course_key__in=course.courserun_set.values('edx_course_key'),
         ).values('order_id').distinct().count()
 
+    def get_custom_number_of_attempts_for_course(self, course):
+        """
+        Get a custom number of exam attempts for given course. If payment was made before
+        the first date, learner get 2 attempts. Otherwise, 1.
+        Args:
+            course (courses.models.Course): a course
+        Returns:
+            int: a number of attempts
+        """
+        lines = Line.objects.filter(
+            order__status__in=Order.FULFILLED_STATUSES,
+            order__user=self.user,
+            course_key__in=course.courserun_set.values('edx_course_key'),
+        ).order_by('created_at')
+        first_date = course.program.exam_attempts_first_date
+        num_attempts = sum([2
```

Comment:
Are the `1` and `2` here related to the `ATTEMPTS_PER_PAID_RUN` constant?

Reply:
yes, i'm going to fix that.